code/modules/awaymissions/mission_code/dsbssigma.dm

```diff
@@ -0,0 +1,735 @@
+	<b>3:</b> Note the gas flow meter to ensure the gas is flowing and enable the pumps along the pipeline leading into the ignition chambers (there should be 2 pumps per side, 1 directly after the mixer and 1 right before the chamber).<br>
+	<b>4:</b> Ensuring that all doors are properly secured you may initiate the ignition by using the button next to the door controller.<br>
+	<b>5:</b> Open the exterior output blast doors so the turbine does not overheat.<br>
+	<b>6:</b> Initiate the turbine boot up procedure on the console and configure the SMES units to evenly match the output.<br><br>
+	Best of luck to you! With me out - you're in charge of keeping this place patched up and running.<br><br>
+	-<i>Quartermaster Lindsey.</i><br>
+	"}
+
+/obj/item/paper/dsbssigma/note5
+	name = "paper - 'Stained Note'"
+	info = {"<br>
+	To whom ever is next on janitorial duty:<br><br>
+	The disposal piping under dormitories appears to have clogged again, if the complaints about the odd smells are anything to go about.<br><br>
+	Flush down some heavy duty cleaner and hope it clears it up so we don't have to bother the engineering staff again.<br><br>
+	-<i>Lindsey.</i><br>
+	"}
+
+/obj/item/paper/dsbssigma/note6
+	name = "paper - 'Torn Report'"
+	info = {"<br>
+	<b>Warden's Report, Cell 6 Incident.</b><br><br>
+	<i>Somehow the prisoner in cell 6 managed to acquire a set of tools and kitchen utensils that they then used to, over the span of several-</i><br>
+	"}
+
+/obj/item/paper/dsbssigma/note7
+	name = "paper - 'Torn Research Report'"
+	info = {"<br><br><br><br>
+	<i>-n further testing, <b>subject 701</b> appears to somehow developed a mutation rendering it highly resistant to viral pathogen. It has been locked up with other specimen, pending transport fo-</i><br>
+	"}
+
+/obj/item/paper/researchnotes/syndicate3
+	name = "paper - 'syndicate research notes'"
+	origin_tech = "syndicate=3"
+	randomized = FALSE
+
+
+/obj/item/organ/internal/heart/gland/heals/weak
+	name = "fleshy growth"
+	desc = "A nausea-inducing growth of flesh."
+	origin_tech = "biotech=5"
+
+/obj/item/organ/internal/heart/gland/heals/weak/activate()
+	to_chat(owner, "<span class='notice'>You feel slightly revitalized.</span>")
+	owner.adjustToxLoss(-10)
+	owner.adjustBruteLoss(-10)
+	owner.adjustOxyLoss(-10)
+	owner.adjustFireLoss(-10)
+
+/obj/item/ammo_casing/c10mm/spent
+	projectile_type = null
+
+/obj/item/ammo_casing/c45/spent
+	projectile_type = null
+
+////// MACHINERY //////
+
+/obj/machinery/computer/id_upgrader/dsbssigma
+	icon_keyboard = "laptop_key"
+	door_to_open = "dsbss_lockdown_blast"
+	upgrade_msg = "<span class='notice'>An access type was added to your ID card. You should be able to lift the lockdown with this.</span>"
+	unlock_msg = "<span class='danger'>WARNING: New user registration during emergency lockdown. Limited lockdown lift in progress.</span>"
+
+/obj/machinery/vending/medical/syndicate_access/dsbssigma
```

Comment:
Does it need inheriting from `/obj/machinery/vending/medical/syndicate_access`? You're overriding `req_access` anyway.